legal-api/src/legal_api/reports/report.py

```diff
@@ -903,6 +901,8 @@ def _format_change_of_registration_data(
                     prev_party_json = VersionedBusinessDetailsService.party_revision_json(
                         prev_completed_filing, prev_party, True
                     )
+                    # current_party type is person, but prev_party type is organization
+                    # so has issue at _has_party_name_change()
                     if self._has_party_name_change(prev_party_json, party):
```

Comment:
`FM1064603`, filing_id: `148571`:
In this case, prev_party is organization. current_party is person.
So `_has_party_name_change` function returns the error. We should fix this logic

Reply:
We can fix this in another ticket.  We need to think about this more when we get back to this work.

I'll create a new ticket.

Can you find a business to test with that doesn't have party type changing?

Reply:
I am trying to find it

Reply:
Ignore my previous comment.  This is a code issue, not a data issue I think.  

You are pulling back an AN record when the code calls `get_party_revision`.

![image](https://github.com/bcgov/lear/assets/6685223/fef8caca-f357-4db7-8300-b964024a0eba)

It should be pulling back the LE person history record.

![image](https://github.com/bcgov/lear/assets/6685223/1999ec31-92b0-4c18-a94c-7f858847d100)

I'm assuming this is related to your `_has_party_name_change` error.

Reply:
Okay, if business.legal_entity_id is null and colin_entity_id is True, then we should pulling back the Colin_entity history?

Reply:
yes, but i'm not entirely sure if that will work as is.  you may need to do some work to get it working.  maybe put a todo and ignore that scenario for now.  just say we need to still take care of that scenario in the future.